biogeochem/EDPatchDynamicsMod.F90

```diff
@@ -2674,43 +2697,149 @@ subroutine terminate_patches(currentSite)
                       call fuse_2_patches(currentSite, youngerPatch, currentPatch)
                       gotfused = .true.
                    endif
+                endif               
+                ! The fusion process has updated the "younger" pointer on currentPatch
+                
+             endif ! older or younder patch
+          endif ! very small area
+
+        else !nocomp. We cannot fuse to patches with a different PFT identity in no competition mode. 
```

Comment:
I guess this is also still placeholder code (since disturbance isn't turned on if nocomp is active for this PR) but I still don't understand why this code below would be required -- why can't you just change the logic on line 2632 (old numbering) from 
`                if (currentPatch%anthro_disturbance_label .eq. olderPatch%anthro_disturbance_label) then`

 to

`                if (currentPatch%anthro_disturbance_label .eq. olderPatch%anthro_disturbance_label .and. 
currentPatch%nocomp_pft_label .eq. olderPatch%nocomp_pft_label) then`

Reply:
So, where the patches are not ordered in any particular way with reference to their PFT, the 'younger' pointer does not necessarily point to a patch of the same PFT and therefore the logic from the anthro_disturbance_label section here won't find the right patch to merge with.

Am I right in thinking that the anthro label patch dynamics code aligned all the primary and secondary patches together in the younger-older hierarchy?  If so, we should/could also enforce that when I re do the dynamic nocomp code? 

I think that it is probably easier for now to delete this code, and bring it back once I've figured out how the multi-patch-per-PFT logic works in spawn_patches.